### PR TITLE
Bug fixes for crash on launch

### PR DESCRIPTION
**Pull request by palu3492**
Fixed two issues. When no msg or attention attribute was sent along with the school's API, it would crash the app. That was fixed by checking those value's type before unwrapping them. Also, fixed alerts so they show when there are no routes.

### CODE REVIEW

File: Linkbus/Linkbus/RouteController.swift [hunk 1]
```diff
@@ -249,8 +249,12 @@ extension RouteController {
         //print(apiBusSchedule.routes?.count)
         //let busSchedule = BusSchedule(msg: apiBusSchedule.msg!, attention: apiBusSchedule.attention!, routes: apiBusSchedule.routes!)
         
-        refreshedLbBusSchedule.msg = csbsjuApiResponse.msg!
-        refreshedLbBusSchedule.attention = csbsjuApiResponse.attention!
+        if(csbsjuApiResponse.msg != nil){
+            refreshedLbBusSchedule.msg = csbsjuApiResponse.msg!
+        }
```
Comment (michaelcarroll) [inline Linkbus/Linkbus/RouteController.swift:254]: Adding 

else { refreshedLbBusSchedule.msg = "" }

Doubt it matters though

File: Linkbus/Linkbus/RouteController.swift [hunk 2]
```diff
@@ -249,8 +249,12 @@ extension RouteController {
...
+        if(csbsjuApiResponse.attention != nil){
+            refreshedLbBusSchedule.attention = csbsjuApiResponse.attention!
+        }
```
Comment (michaelcarroll) [inline Linkbus/Linkbus/RouteController.swift:257]: else { refreshedLbBusSchedule.attention = "" }

File: Linkbus/Linkbus/RouteController.swift [hunk 3]
```diff
@@ -269,7 +273,7 @@ extension RouteController {
             refreshedLbBusSchedule.alerts.append(dailyMessageAlert)
         }
         
-        if !(csbsjuApiResponse.routes!.isEmpty) {
+        if (!csbsjuApiResponse.routes!.isEmpty) {
```
Comment (michaelcarroll) [inline Linkbus/Linkbus/RouteController.swift:276]: This didn't actually change anything right? Just preference?

File: Linkbus/Linkbus/RouteController.swift [hunk 4]
```diff
@@ -394,11 +398,12 @@ extension RouteController {
                     tempRoute.nextBusTimer = nextBusTimer
                     
                     refreshedLbBusSchedule.routes.append(tempRoute)
-                    lbBusSchedule = refreshedLbBusSchedule
                 }
             }
         }
         
+        lbBusSchedule = refreshedLbBusSchedule
```
Comment (michaelcarroll) [inline Linkbus/Linkbus/RouteController.swift:406]: Good catch

**Comment by michaelcarroll** [comment]
Thanks

**Comment by michaelcarroll** [comment]
Tested changes and app behaves as expected with or without busses scheduled. Pushing to App Store.

File: Linkbus/Linkbus/RouteController.swift (hunk 1 above)
Comment (palu3492) [reply Linkbus/Linkbus/RouteController.swift:254]: I don't think it matters. Might do that by default?

File: Linkbus/Linkbus/RouteController.swift (hunk 3 above)
Comment (palu3492) [reply Linkbus/Linkbus/RouteController.swift:276]: Accident I suppose. You can't do that in JavaScript so it hurts my brain to see.

**Pull request merged**